UnityProject/Assets/Scripts/Managers/CargoManager.cs

```diff
@@ -59,6 +59,23 @@ private void OnDisable()
 		SceneManager.activeSceneChanged -= OnRoundRestart;
 	}
 
+	/// <summary>
+	/// Checks for any Lifeforms (dead or alive) that might be aboard the shuttle
+	/// </summary>
+	/// <returns>true if a lifeform is found, false if none is found</returns>
+	bool CheckLifeforms()
+	{
+		Transform ObjectHolder = CargoShuttle.Instance.SearchForObjectsOnShuttle();
+		foreach (Transform child in ObjectHolder)
+		{
+			if (child.transform.gameObject.layer == 12 || child.transform.gameObject.layer == 8)
```

Comment:
Can't you just do `child.gameObject.layer`? Because `child` is already `transform`.

Reply:
Cache `LayerMask` before `foreach`
`LayerMask layersToCheck = LayerMask.GetMask("Player", "NPC")`

Then do the check `if((child.gameObject.laer & layersToCheck) == 0) continue`
`== 0` will check if it's false. *Or i might be wrong and it's `!= 0`*